apps/prairielearn/src/lib/db-types.ts

```diff
@@ -441,6 +441,23 @@ export const EnrollmentSchema = z.object({
 });
 export type Enrollment = z.infer<typeof EnrollmentSchema>;
 
+export const FileSchema = z.object({
+  assessment_id: IdSchema.nullable(),
+  assessment_instance_id: IdSchema.nullable(),
+  created_at: DateFromISOString,
+  created_by: IdSchema.nullable(),
+  deleted_at: DateFromISOString.nullable(),
+  deleted_by: IdSchema.nullable(),
+  display_filename: z.string(),
+  id: IdSchema,
+  instance_question_id: IdSchema.nullable(),
+  storage_filename: z.string(),
+  storage_type: z.enum(['FileSystem', 'S3']),
+  type: z.string(),
+  user_id: IdSchema.nullable(),
+});
+export type File = z.infer<typeof FileSchema>;
```

Comment:
Is it a problem that this overlaps with the built-in File type?

Reply:
Great question. @nwalters512 do you have an input on this one?

Reply:
This is definitely less than ideal, but I think we can live with this. If we ever end up in a situation where we need both in one file, we an always either alias with `import { File as DatabaseFile }` or rename it entirely.